docs/user-guide/misc/visualization.md

```diff
@@ -0,0 +1,65 @@
+# Visualization
+
+Data in a Polars `DataFrame` can be visualized using common visualization libraries.
+
+<div style="display:none">
+```python exec="on" result="text" session="user-guide/misc/visualization"
+--8<-- "python/user-guide/misc/visualization.py:setup"
+```
+</div>
+
+We illustrate plotting capabilities using the Iris dataset. We scan a CSV and then do a group-by on the `species` column and get the mean of the `petal_length`.
+
+{{code_block('user-guide/misc/visualization','dataframe',[])}}
+
+```python exec="on" session="user-guide/misc/visualization"
+--8<-- "python/user-guide/misc/visualization.py:dataframe"
+```
+
+### Built-in plotting with hvPlot
```

Comment:
should this one be `##`? I'd make it at least as visible as matplotlib

Reply:
Yep, done